tests/test_hysr_control.py

```diff
@@ -220,3 +305,141 @@ def test_instant_reset(run_pam_mujocos, hysr_control_instance):
     )
     for eb1, eb2 in zip(reset_states.extra_balls, initial_states.extra_balls):
         assert eb1.joint_positions != pytest.approx(eb2.joint_positions, 0.01)
+
+
+def test_hysr_control_factory(run_pam_mujocos, get_factory_classes):
+    """
+    Test hysr_control_factory succeed in returning an instance of HysrControl.
+    """
+
+    algo_time_step = 0.01
+    mujoco_time_step = 0.002
+
+    pressure_robot, main_sim, extra_balls = get_factory_classes
+
+    hysr_control: hysr.Hysr_control = (  # noqa: F841 (variable not used)
+        hysr.hysr_control_factory(
+            pressure_robot, main_sim, (extra_balls,), mujoco_time_step, algo_time_step
+        )
+    )
```

Comment:
I'm a bit confused what is tested here.  Shouldn't there be some check if `hysr_control` is good in the end?

Reply:
this is just testing no exception are thrown